docs/reports/prometheus-collector.md

```diff
@@ -0,0 +1,18 @@
+# Prometheus Collector
+
+The Prometheus Collector report gathers workload metrics from a Prometheus installation
+in order to provide fine-grained resource usage data. This can be used to gauge how much
+different workloads cost, understand cost trends, and help set resource requests and limits.
+
+## Use an existing Prometheus installation
+To use an existing Prometheus installation pass the following flag to the Helm chart installation:
```

Comment:
Should we give them the UI instructions instead of telling them to modify the helm parameters themselves?

Reply:
I think this is simpler. Hopefully soon we can get the UI pulling from the reported values so the two stay in line